pkg/kubectl/cmd/exec.go

```diff
@@ -161,6 +162,10 @@ func (p *ExecOptions) Complete(f *cmdutil.Factory, cmd *cobra.Command, argsIn []
 		}
 	}
 
+	if strings.HasPrefix(p.PodName, "pod/") {
```

Comment:
@kargakis thanks, looked at it and thought it's a bit overkill, yet it seems to be a proper way here